CMakeLists.txt

```diff
@@ -3,4 +3,7 @@
 cmake_minimum_required(VERSION 2.6)
 find_package(Rock)
 rock_init(dvl_teledyne 0.1)
+
+SET( CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -std=c++0x" )
```

Comment:
Use CMake's support for C++11/C++14 please. See https://rock-core.github.io/rock-and-syskit/integrating_functionality/cpp_libraries.html#cxx_standard

and in base/cmake: https://github.com/rock-core/base-cmake/blob/master/modules/Rock.cmake#L9